microapps/ArTabletopApp/app/src/main/java/com/arcgismaps/toolkit/artabletopapp/screens/MainScreen.kt

```diff
@@ -74,6 +75,15 @@ fun MainScreen() {
             }
         }
     }
+    // disable pan/zoom/rotate interaction
```

Comment:
I think it might be worth adding a brief explanation here, ie.

```suggestion
    // disable pan/zoom/rotate interaction. These interactions can behave unexpectedly in TableTop scenarios
```